documentation/tutorial/01-introduction.md

```diff
@@ -0,0 +1,24 @@
+# Charmed Superset
+
+This guide describes how you can get started with Charmed Superset. From setting up MicroK8s in your host environment, all the way to creating your first dashboard.
+
+Charmed Superset delivers automated operations management from day 0 to day n for [Apche Superset](https://superset.apache.org/). It is an open source, end-to-end, production-ready data visualization tool on top of [Juju](https://juju.is/).
+
+
+## Prerequisites
+
+- Ubuntu 22.04 LTS or later.
+- [Snap](https://snapcraft.io/) installed.
+- 8GB of RAM.
+- 2 CPU threads.
+- At least 20GB of available storage.
+- Access to the internet for downloading the required rocks and charms.
```

Comment:
Is this one needed? It seems to be obvious nowadays, had the same question in other Canonical products

Reply:
Probably not, it's unlikely anyone will deploy superset to an air-gapped environment. I'll remove it.